examples/add_pedestal_noise_template.xml

```diff
@@ -0,0 +1,114 @@
+<container>
```

Comment:
could you add a doc string that is telling what the process' purpose and what it is doing?

Reply:
Done.

Reply:
placet